symbol.md

```diff
@@ -0,0 +1,181 @@
+# SYMBOLS#PROCS
+
+Thank you for accepting this mission, and the mission is nothing else but to add Symbols#procs to our Ruby toolchain . However, to achieve our mission, we need to have expertise of  **symbols** & **Procs** and then with their combined power we will take on the mission.
+
+If you have some difficulties with any of these concepts, do not worry, I made arrangements for you. [The Ruby_Newbie Guide to Symbols](http://www.troubleshooters.com/codecorn/ruby/symbols.htm), has exactly what you will need for symbols and [Ruby Explained: Blocks, Procs, and Lambdas, aka "Closures"](http://www.eriktrautman.com/posts/ruby-explained-blocks-procs-and-lambdas-aka-closures). Once you are well equipped, onward to the mission.
+
+Let's construct a class for our soldier object
+```ruby
+  class Soldier
+    def initialize(name, weapon)
+      @name = name
+      @weapon = weapon
+    end
+
+    def name
+      @name
+    end
+
+    def first_name
+      @name.split.first
+    end
+
+    def last_name
+      @name.split.last
+    end
+
+    def weapon
+      @weapon
+    end
+
+    def salary
+      500 * name.length
+    end
+
+    def grade_a_weapon?
+      weapon.length > 5
+    end
+  end
+
+  soldier = Soldier.new("John Doe", "AK-47")
+
+  # From our symbol armory
+  soldier.send(:name) #=> John Doe
```

Comment:
The reason for it's use here (and thereafter) is to make users comfortable with using the `send` methods. I was trying to avoid introducing them _sharply_ to it's use in the implementation of `:to_proc`, which might cause my having to explain with examples the `:send` method does.